code/modules/admin/NewBan.dm

```diff
@@ -92,11 +92,23 @@ var/savefile/Banlist
 
 	return 1
 
-
+/proc/add_banfile(var/client/c)
+	var/savefile/file = new()
+	file["key"] << c.ckey
+	file["ip"] <<c.address
+	file["cid"] <<c.computer_id
+	c.Export(file)
+/proc/remove_banfile(var/client/c)
+	c.Export()

```

Comment:
Straight from the docs "When a file is exported to the player's computer, it replaces any previous file stored by a game with the same world.hub value. This should not be used for anything requiring high security, because any other world could make use of the same hub path and access the file. It is also possible for the user to tinker with the file, since it resides on their computer."